### ISSUE TITLE
Soft button content is misaligned in an alert 

### ISSUE THREAD

**Issue by NicoleYarroch** [smartdevicelink/generic_hmi#323]
## Bug Report
If a soft button in an alert has both images and text, the text extends outside the bounds of the button. This was seen on Firefox web browser (Mozilla Firefox 83.0)

### Reproduction Steps
1. Create an alert with two soft buttons. One button is text-only ("Ok") and the second button has text and an image ("Gotcha" with the speech bubble icon). 
1. Send the alert request. 

#### Expected Result
The soft button text and image is center aligned inside the button.

#### Actual Result
The soft button text and image are misaligned. Text extends outside the bounds of the soft button. 

<img width="963" alt="Screen Shot 2020-11-18 at 2 05 46 PM" src="https://user-images.githubusercontent.com/19915981/99577252-14328880-29a9-11eb-9fa9-336441abb5c3.png">

**Comment by iCollin** [comment]
Hello @NicoleYarroch, could you please share with me the exact RPC you were sending when you saw this issue? I haven't been able to reproduce with a similar setup.

Also is this your exact output?
```
$ firefox --version
Mozilla Firefox 83.0
```

Thank you!

btw, this is what I was sending:
```
{
    "rpc":"Alert",
    "parameters":{
        "alertText1":"goodjob!",
        "softButtons":[
            {
                "type":"TEXT",
                "softButtonID":1,
                "text":"sb1"
            },
            {
                "type":"BOTH",
                "softButtonID":2,
                "text":"gotcha!",
                "image":{
                    "value":"0xCD",
                    "imageType":"STATIC"
                }
            }
        ]
    }
}
```

**Comment by NicoleYarroch** [comment]
I was still able to reproduce this on the **develop** branch of generic_hmi.
Firefox version: Mozilla Firefox 83.0
Generic_hmi commit: 2fad670de204cd2cce094a0a9dc71630a37a6911 (HEAD -> develop, origin/develop)
SDL_core commit: 5c72cddc353aadef06e89064c53c1ec6b2467d25 (HEAD -> release/7.0.0, tag: 7.0.0-RC1)

<img width="957" alt="Button Bug" src="https://user-images.githubusercontent.com/19915981/102089725-9c5e4f00-3dea-11eb-8a28-f12f9e512080.png">

This is what I sent for the image above:
```
2020-12-14 08:55:39.356759-0500 SDL_Smoke_Tests[22767:679897] [Protocol] Sending RPC: Alert (request), id: 24
{
    alertIcon = "{\n    imageType = DYNAMIC;\n    isTemplate = 0;\n    value = a815096a889d6987;\n}";
    alertText1 = "second alert";
    alertText2 = 2;
    alertText3 = 3;
    cancelID = 8;
    duration = 5000;
    playTone = 0;
    progressIndicator = 0;
    softButtons =     (
        "{\n    image = \"{\\n    imageType = DYNAMIC;\\n    isTemplate = 1;\\n    value = e9793f97e11cf753;\\n}\";\n    isHighlighted = 0;\n    softButtonID = 1;\n    systemAction = \"DEFAULT_ACTION\";\n    text = \"Button 1\";\n    type = BOTH;\n}",
        "{\n    image = \"{\\n    imageType = DYNAMIC;\\n    isTemplate = 1;\\n    value = e9793f97e11cf753;\\n}\";\n    isHighlighted = 0;\n    softButtonID = 101;\n    systemAction = \"DEFAULT_ACTION\";\n    text = \"Button 2\";\n    type = BOTH;\n}"
    );
}
```

**Comment by iCollin** [comment]
@NicoleYarroch I was able to reproduce using dynamic images and I believe I have fixed the issue in #336.

**Comment by jacobkeeler** [comment]
Closing with the merge of #338
